asQ/allatoncesystem.py

```diff
@@ -106,6 +275,9 @@ def __init__(self,
         self.form_mass = form_mass
         self.form_function = form_function
 
+        self.linearised_mass = linearised_mass if linearised_mass is not None else form_mass
+        self.linearised_function = linearised_function if linearised_function is not None else form_function
```

Comment:
I think the best names are
linearised_mass -> form_mass_J
linearised_function -> form_function_J